spec/models/category_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+describe Category do
+  it { should have_many(:videos)}
+
+  describe "recent_videos" do
```

Comment:
You should stick to RSpecs style on how to annotate `describe` blocks - check here http://betterspecs.org/#describe